app-web/src/components/Form/form.js

```diff
@@ -0,0 +1,87 @@
+import React, { Fragment } from 'react';
+import styled from '@emotion/styled';
+import { Field } from 'react-final-form';
+
+// Styled components ....
+
+export const StyledErrorMessage = styled.div`
+  font-size: 12px;
+  color: red;
+  width: 400px;
+  margin-top: 0.25rem;
+  &:before {
+    content: '❌ ';
+    font-size: 10px;
+  }
+  @media (prefers-color-scheme: dark) {
+    color: red;
+  }
+`;
+
+export const StyledLabel = styled.label`
+  margin-top: 1rem;
+  display: block;
+`;
+
+export const StyledButton = styled.button`
+  position: relative;
+  width: 150px;
+  height: 36px;
+  border-radius: 2px;
+  margin: 20px 2px;
+  background-color: white;
+  color: black;
+  cursor: pointer;
+`;
+
+export const SubmitButton = styled.button`
```

Comment:
lets go over this css later on. Approving  for now. please  leave a mental note to go over  this.

Reply:
also we have a  Button component. Please take a look at `/UI`